x-pack/plugins/maps/public/classes/layers/layer.tsx

```diff
@@ -92,7 +93,8 @@ export interface ILayer {
   isVisible(): boolean;
   cloneDescriptor(): Promise<LayerDescriptor>;
   renderStyleEditor(
-    onStyleDescriptorChange: (styleDescriptor: StyleDescriptor) => void
+    onStyleDescriptorChange: (styleDescriptor: StyleDescriptor) => void,
+    onCustomIconChange: (customIcons: CustomIcon[]) => void
```

Comment:
change to onCustomIconsChange so its the same as in implementation and shows up in search results